diagnostic.md

```diff
@@ -184,5 +211,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+The element would be 25px if the _screen width_ were 550px.
```

Comment:
Since the screen width is 550px, the `@media(min-width= 400px)` is applied. So `.stretch` is 25%. Thus 25% * 550px is 137.50px